assets/app/view/game/spreadsheet.rb

```diff
@@ -156,6 +156,32 @@ def render_or_history_row(hist, corporation, x)
         end
       end
 
+      def render_connection_run(corporation)
+        return [] unless @game.respond_to?(:connection_run)
+        return [h(:td)] unless @game.connection_run[corporation]
+
+        x, c_run = @game.connection_run[corporation]
+        revenue_text, alpha =
+          case (c_run.dividend.is_a?(Engine::Action::Dividend) ? c_run.dividend.kind : 'withhold')
+          when 'withhold'
+            ["[#{c_run.revenue}]", '0.5']
+          else
+            [c_run.revenue.to_s, '1.0']
+          end
+
+        props = {
+          style: {
+            color: convert_hex_to_rgba(color_for(:font2), alpha),
+          },
+        }
+
+        link_h = history_link(revenue_text,
+                              "Go to connection run of #{corporation.name} (in #{x})",
```

Comment:
`x.join('.')` would fit the style used everywhere else.
(bonus points if you fix L147, too ;-) )
Don’t use "x" if there are better var names available.

Reply:
Again, inspired on render_or_history_row doing the same thing. Will try to use the other style.